commands.js

```diff
@@ -2476,8 +2476,11 @@ exports.commands = {
 		if (!this.can('hotpatch')) return false;
 		if (Monitor.hotpatchLock) return this.errorReply("Hotpatch is currently been disabled. (" + Monitor.hotpatchLock + ")");
 
-		let staff = Rooms('staff');
-		if (staff) staff.add("(" + user.name + " used /hotpatch " + target + ")").update();
+		let announceRooms = ['development', 'staff', 'upperstaff'];
+		for (let i in announceRooms) {
```

Comment:
You can just do `for (let roomid of ['development', 'staff', 'upperstaff']) {`

You shouldn't do `for..in` on arrays; that's one of the things `CONTRIBUTING.md` explicitly tells you not to do iirc.